builtin/rebase.c

```diff
@@ -536,6 +537,9 @@ static int run_am(struct rebase_options *opts)
 	am.git_cmd = 1;
 	argv_array_push(&am.args, "am");
 
+	if (opts->ignore_whitespace)
+		argv_array_push(&am.args, "--ignore-whitespace");
```

Comment:
I was worried at first when reading this that pushing the flag onto am.args forces the rebase to use the am-backend, but this is in run_am() meaning that you've already gone down that route.  So this looks good.